docs/overview.md

```diff
@@ -154,34 +154,44 @@ Once supermajority is reached, Alice submits `unlock` transaction to the EVM cha
 
 The transfer of NFT assets from Hedera to the EVM chain is described in the following sequence diagram.
 <p align="center">
-  <img src="./assets/hedera-to-evm(nft%20hedera%20native).png">
+  <img src="./assets/hedera-to-evm%28nft%20hedera%20native%29%20using%20crypto%20allowances.png">
 </p>
 
 #### Steps
 1. **Initiating the transfer**
-   Alice wants to transfer `HTS NFT` from Hedera to the EVM chain. She opens any UI that integrates the Bridge and sends the NFT asset and flat fee to the `Bridge` Account. The memo of the transfer contains the `evm-address`, which is going to be the receiver of the wrapped ERC-721 (NFT) asset on the other chain.
-2. **Picking up the Transfer**
-   The Bridge validator nodes listen for new incoming transfers to the `Bridge` Account. Once they pick up the new transaction, they validate that the `memo` contains a valid EVM address configured as receiver of the wrapped asset.
-3. **Paying out fees**
+   
+   **1.1** Alice sends `Allowance` for the NFT to the `Payer Account` of the `Bridge`
+   
+   **1.2** Alice wants to transfer `HTS NFT` from Hedera to the EVM chain. She opens any UI that integrates the Bridge and sends the flat fee and all the necessary Royalty (fallback fee) and Fixed fees for the NFT asset to the `Bridge` Account. The memo of the transfer contains the other `chain-Id`, the `evm-address` which is going to be the receiver of the wrapped ERC-721 (NFT) asset and the Hedera NFT ID `serial@token-id`.
```

Comment:
`Alice wants to transfer HTS NFT from Hedera to the EVM chain. She opens any UI that integrates the Bridge` - this has to be before 1.1.